src/ClientData/include/ClientData/CallstackData.h

```diff
@@ -75,6 +76,19 @@ class CallstackData {
     }
   }
 
+  // Do a particular action for callstacks but skipping callstacks that will be rendered later in
+  // the same pixel in the screen. It assures to do the action at least once for callstacks in each
+  // occupied pixel. This iteration is faster that the non-discretized one since it doesn't require
```

Comment:
"at least once per callstack" suggests the same action could be done more than once per callstack? That doesn't sound like what we want?

Reply:
done